src/atem.ts

```diff
@@ -172,7 +172,7 @@ export class BasicAtem extends EventEmitter<AtemEvents> {
 		const allChangedPaths: string[] = []
 
 		const state = this._state
-		commands.forEach((command) => {
+		commands.forEach(async (command) => {
```

Comment:
I think this is better done as a for loop now because using async callbacks in a forEach may create race conditions.